assets/src/web-stories-block/css/core-themes/twentytwentyone.css

```diff
@@ -14,10 +14,35 @@
   }
 
   /* Fix for menu toggle overlap. */
-  .web-stories-theme-header-section .web-stories-list--customizer {
+  .web-stories-theme-header-section .web-stories-list {
     margin-top: 72px;
   }
 
+  .web-stories-theme-header-section .web-stories-list.is-view-type-circles {
+    margin-top: 72px;
+    padding-top: 0;
+  }
+
+  .web-stories-theme-header-section
+    .web-stories-list.is-carousel.has-archive-link {
+    margin-top: 24px;
+    padding-top: 72px;
+  }
+
+  /* Fix for menu toggle overlap on AMP. */
+  html[amp] .web-stories-theme-header-section .web-stories-list.is-carousel {
+    margin-top: 0;
+    padding-top: 72px; /* same as above */
+  }
+
+  html[amp]
+    .web-stories-theme-header-section
+    .web-stories-list.is-carousel.has-archive-link {
+    padding-top: calc(
+      72px + 24px
+    ); /* default margin from common.css plus above offset */
```

Comment:
Why is this a calc and not just 96?

Reply:
This coding style is just copied from elsewhere in the block stylesheets for consistency.
IMO this way makes it clearer how the final value is assembled (rather than just writing `96px`).
Plus during build this gets automatically converted to `padding-top: 96px` anyway, so causes no harm.